pkg/drain/podDeleteStrategy.go

```diff
@@ -42,8 +43,16 @@ func (pds *podDeletionStrategy) IsValid(node *corev1.Node) (bool, error) {
 }
 
 func (pds *podDeletionStrategy) getPodList(node *corev1.Node) (*corev1.PodList, error) {
```

Comment:
Anyway we can refactor this conveniently to avoid DRY? 
https://github.com/openshift/managed-upgrade-operator/pull/264/files#diff-660b7891a9e10f0f7ba55d478514fd0f6c320bd12948cb0f183fafd7907cf02cR44 
has exact same method sig/body bar the type whos method it is.

Reply:
same thing https://github.com/openshift/managed-upgrade-operator/pull/264/files#diff-0b625a2c601056a44ac3c725ea7e19ad99151fce34e34973aac4badb03cdc7b9R45

Reply:
This makes sense - we'll follow this up in OSD-7882